docs/ref/forms/fields.txt

```diff
@@ -1228,6 +1228,16 @@ method::
 
         Same as :class:`ModelChoiceField.to_field_name`.
 
+.. method:: ModelMultipleChoiceField.validate_choices(qs, key, values)
+
```

Comment:
Missing `.. versionadded:: 2.0` annotation and a mention in `docs/releases/2.0.txt` -- please use the [patch review checklist](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#patch-review-checklist).

Perhaps some discussion of the use cases would be useful. For example, it's not clear to me if putting `qs = self.queryset.filter(**{'%s__in' % key: value})` in the new method would be useful. Do you expect someone to call `super()` in their own `validate_choices()` or not?